submissions/Semka094/hooli-style-popup/style.css

```diff
@@ -0,0 +1,244 @@
+* {
+    font-family: Jost, Helvetica Neue, Arial, sans-serif;
+    margin: 0;
+    padding: 0;
+}
+
+.hiddenItems {
+    display: none;
+}
+
+.popupContainer {
+    width: 400px;
+    border: 1px solid rgba(173, 139, 156, 0.34);
+    border-radius: 5px;
+    box-shadow: -3px 10px 23px 1px rgba(242, 220, 232, 0.22);
+    overflow: scroll;
+    height: 405px;
+    position: relative;
+}
+
+.popupContainer:after {
+    content: "";
+    width: 15px;
+    height: 15px;
+    transform: rotate(-45deg);
+    background: #fff;
+    position: absolute;
+    box-shadow: 2px -2px 2px -1px rgb(242 220 232);
+    z-index: 0;
+    top: -5px;
+    left: calc(90% - 3px);
+}
+
+.menuContainer {
+    background-color: #f5f7fa;
+    height: 65px;
+    display: flex;
+    align-items: center;
+    justify-content: right;
+    padding-right: 50px;
+}
+
+.navWrapper {
+    display: flex;
+    align-items: center;
+    height: 50px;
+}
+
+.navWrapper .navItem {
+    border: 2px solid transparent;
+    position: relative;
+    padding: 5px 8px;
+}
+
+.navWrapper .navItem:hover {
+    border: 2px solid #f6ccdb;
+    border-radius: 3px;
+}
+
+.navWrapper .navItem .navLink {
+    font-size: 18px;
+    color: dimgray;
+    text-decoration: none;
+    height: inherit;
+    display: block;
+}
+
+.navWrapper .navItem img {
```

Comment:
Avoid styling by HTML tags except if you add base styles. [Reason](https://frontstuff.io/you-need-to-stop-targeting-tags-in-css) Also, avoid too specific CSS selectors. [Read more about that](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity)